src/app/base/entities/project.ts

```diff
@@ -7,4 +7,6 @@ export interface Project {
   projectType: string;
   description?: string;
   numDataScaleUploaded?: number;
+  tokenizer?: string;
+  containsUniqueAttribute?: boolean;
```

Comment:
ok this one took me a bit :D i didn#t know what this is meant to be, especially since it's never used (only requested for the project by project id request)

since we don't use it in the frontend atm I would just remove it from the query and here. 

In general we should rename the field to something like has_primary_key in the backend (at least that is whats check) -> open an issue on the gateway to rename the field